lib/spack/llnl/util/filesystem.py

```diff
@@ -1132,6 +1248,10 @@ def find(root, files, recursive=True):
         return _find_non_recursive(root, files)
 
 
+# here and in _find_non_recursive below we only take the first
+# index to check for system path safety as glob handles this
+# w.r.t. search_files
```

Comment:
Does this comment still apply? The function is no-longer changed other than the use of `system_path_filter`